src/utils/vr-caps-detect.js

```diff
@@ -23,8 +23,10 @@ function isMaybeDaydreamCompatibleDevice(ua) {
 const GENERIC_ENTRY_TYPE_DEVICE_BLACKLIST = [/cardboard/i];
 
 export function detectInHMD() {
-  const isOculusBrowser = /Oculus/.test(navigator.userAgent);
-  return isOculusBrowser;
+  const ua = navigator.userAgent;
+  const isFirefoxReality = /Firefox/.test(ua) && /Android/.test(ua) && window.hasNativeWebVRImplementation;
```

Comment:
I think this ends up getting xr-polyfilled to always be `true`, oddly enough. You should confirm but I think this will regress us back to thinking firefox android is a VR capable browser. The proper check is what you have here but also for "VR" as well I think, as per the latest update for 1.1.